app/models/manageiq/providers/inventory/persister/builder/automation_manager.rb

```diff
@@ -43,10 +43,6 @@ def inventory_root_groups
           add_common_default_values
         end
 
-        def vms
-          add_properties(:manager_ref => %i[uid_ems])
-        end
```

Comment:
This was used by ansible_tower to cross-link vms, interestingly this was the only property they added to the core definition and continued to define the rest of the attributes in the provider [[here]](https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/272/files#diff-4949e15a250d6715677f53bc2c31a7e36a634a7b85529019198fe64ec1b78574L36-L43)

Reply:
Does removing this mean I have to merge the Ansible tower PR with this one?  Or can I merge this one first and do the others after?

Reply:
Yes it would mean they are codependent, I'll drop this change out and we can do this as a follow-up then this can be merged independently.